libraries/ilo_helper.rb

```diff
@@ -10,33 +10,36 @@
 # specific language governing permissions and limitations under the License.
 
 # Module to load the ilo-ruby-sdk gem and build the client object
-module IloHelper
-  # Load (and install if necessary) the ilo-sdk
-  def load_sdk
-    gem 'ilo-sdk', node['ilo']['ruby_sdk_version']
-    require 'ilo-sdk'
-    Chef::Log.debug("Found gem ilo-sdk (version #{node['ilo']['ruby_sdk_version']})")
-  rescue LoadError
-    Chef::Log.info("Did not find gem ilo-sdk (version #{node['ilo']['ruby_sdk_version']}). Installing now")
-    chef_gem 'ilo-sdk' do
-      version node['ilo']['ruby_sdk_version']
-      compile_time true if Chef::Resource::ChefGem.method_defined?(:compile_time)
+module IloCookbook
+  ## Helper module
+  module IloHelper
```

Comment:
Also, since this is now scoped inside the `IloCookbook` module, we can probably rename it to just `Helper`